File: v6/lib/commands/serverc.dart
import 'package:bad_word_blocker_dart/globals.dart';
import 'package:bad_word_blocker_dart/types.dart';
import 'package:nyxx/nyxx.dart';
import 'package:nyxx_interactions/nyxx_interactions.dart';

Future<void> serverCCommandHandler(
    ISlashCommandInteractionEvent event, Guild guild) async {
  final embed = EmbedBuilder()
    ..title = "Server Count"
    ..description = "Bad Word Blocker is in ${client.guilds.length} servers."
    ..color = blueEmbedColor;
  await event.respond(MessageBuilder.embed(embed), hidden: true);
}

File: v5/Handlers/Commands/Serverc.fs
module Handlers.Commands.Serverc

open Discord.WebSocket
open GlobalVariables
open System.Threading.Tasks
open Utils
open Wrappers.DiscordWrappers
open Types

let servercCommand (commandInteraction: SocketSlashCommand) =
  task {

    let guildCount = getGuildCount client

    do!
      send
        (Interaction(commandInteraction, Some true))
        { MessageType =
            Embed
              { Title = "Server count"
                Description = $"Bad Word Blocker is in {guildCount} servers"
                Footer = None
                Color = Blue }
          Components = None }
      :> Task
  }
